collection/lens.rkt

```diff
@@ -39,6 +67,21 @@
   (make-lens (λ (target) (nth target index))
              (λ (target value) (set-nth target index value))))
 
+(define (mapper-lens lens)
```

Comment:
I think all lenses are implicitly nouns. For example, this library has `ref-lens`, not `reffer-lens`. Lenses seem to have been named based on the action they represent, which in this case would be `map`.